service/src/main/java/org/eclipse/keti/acs/policy/evaluation/cache/PolicyEvaluationRequestCacheKey.java

```diff
@@ -125,6 +126,13 @@ public Builder request(final PolicyEvaluationRequestV1 request) {
             return this;
         }
 
+        public Builder policySetId(final String policySet) {
+            if (null != policySet) {
+                this.policySetIds(Stream.of(policySet).collect(Collectors.toCollection(LinkedHashSet::new)));
```

Comment:
why create a Stream of a single value?
` this.policySetIds(new LinkedHashSet<>(Collections.singleton(policySet)));`